plugins/modules/dns_record.py

```diff
@@ -0,0 +1,591 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: dns_record
+short_description: Manage Record
+description:
+    - Manage Record
+version_added: 2.0.0
+author: Infoblox Inc. (@infobloxopen)
+options:
+    id:
+        description:
+            - ID of the object
+        type: str
+        required: false
+    state:
+        description:
+            - Indicate desired state of the object
+        type: str
+        required: false
+        choices:
+            - present
+            - absent
+        default: present
+    absolute_name_spec:
+        description:
+            - "Synthetic field, used to determine I(zone) and/or I(name_in_zone) field for records."
+        type: str        
+    comment:
+        description:
+            - "The description for the DNS resource record. May contain 0 to 1024 characters. Can include UTF-8."
+        type: str
+    disabled:
+        description:
+            - "Indicates if the DNS resource record is disabled. A disabled object is effectively non-existent when generating configuration."
+            - "Defaults to I(false)."
+        type: bool
+    inheritance_sources:
+        description:
+            - "The inheritance configuration specifies how the Record object inherits the ttl field"
+        type: dict
+        suboptions:
+            ttl:
+                description: "The inheritance configuration for a field of type UInt32."
```

Comment:
```suggestion
                description: "The TTL inheritance settings for the record. If not set, the record will inherit the TTL from the zone."
```
Please change all occurrences.

Reply:
Fixed